tests/storage/storage.go

```diff
@@ -1437,3 +1437,12 @@ func checkResultShellCommandOnVmi(vmi *v1.VirtualMachineInstance, cmd, output st
 	ExpectWithOffset(1, res).ToNot(BeEmpty())
 	ExpectWithOffset(1, res[0].Output).To(ContainSubstring(output))
 }
+
```

Comment:
@ormergi Hi, I already did it as part of #11223

Reply:
I removed this change